action.yml

```
@@ -194,6 +198,7 @@ runs:
     run: |
       ACTION_WORKSPACE=$(echo "${GITHUB_WORKSPACE}" | grep -Eo "^.*work")
       docker run \
+        $([ "${{ inputs.privileged }}" = "true" ] && echo "--privileged") \
```

```suggestion
        $([ "${{ inputs.privileged }}" = "true" ] && echo -n "--privileged") \
```

Otherwise it adds a newline which we don't want I think